From ef0004dcf517ab1d39f944a7e6bca969abf3f9a8 Mon Sep 17 00:00:00 2001
From: Andrew Svetlov <andrew.svetlov@gmail.com>
Date: Sat, 23 Sep 2017 12:37:48 +0300
Subject: [PATCH] Make tests more stable

---
 tests/test_istr.py | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/tests/test_istr.py b/tests/test_istr.py
index e5539c2..430870a 100644
--- a/tests/test_istr.py
+++ b/tests/test_istr.py
@@ -1,6 +1,7 @@
 from multidict._multidict import istr
 from multidict._multidict_py import istr as _istr
 import gc
+import sys
 import psutil
 
 
@@ -70,7 +71,12 @@ class TestPyIStr(IStrMixin):
         gc.collect()
         info2 = p.memory_info()
         rss_diff = info2.rss - info.rss
-        assert rss_diff == 0
+        if sys.platform == 'win32':
+            # on windows one page could be not returned
+            # to OS
+            assert rss_diff <= 4096
+        else:
+            assert rss_diff == 0
 
 
 class TestIStr(IStrMixin):
-- 
2.14.1

